tests/compute/test_transform.py

```diff
@@ -1013,10 +1024,11 @@ def test_cast():
     # test_remove_self_loop()
     # test_add_self_loop()
     # test_partition_with_halo()
-    # test_metis_partition()
+    test_metis_partition()
+    test_hetero_metis_partition()
     # test_hetero_linegraph('int32')
     # test_compact()
-    test_to_simple("int32")
+    # test_to_simple("int32")
```

Comment:
why comment out the tests?

Reply:
It's just for debug usage. Pytest will run all functions with name starting with "test"